.gitignore

```diff
@@ -6,6 +6,25 @@ src/libfaketime.dylib.1
 src/libfaketime.1.dylib
 src/core
 src/faketime
-m4/*.m4*
 
 .vscode/
+
+Makefile
```

Comment:
Why are we version-controlling the `Makefile`? This should be generated by running the `configure` script.

Reply:
This all looks good, though I don't think you need all of the files you added to git to be version-controlled.

Reply:
This is the `.gitignore` file; it lists files that, as you say, should be not version-controlled.